### PR TITLE
Updated readme by providing the correct link to the example_scenes

### PR DESCRIPTION
**Pull request by nidhaloff**
update the link to the "example_scenes" page

## Checklist
- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [ ] I have written a descriptive PR title (see top of PR template for examples)
- [ ] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [x] The PR title is descriptive enough
- [x] The PR is labeled correctly
- [x] The changelog entry is completed if necessary
- [x] Newly added functions/classes either have a docstring or are private
- [x] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ manim -p -ql example.py SquareToCircle
 ```
 
 You should see your native video player program pop up and play a simple scene in which a square is transformed into a circle. You may find some more simple examples within this
-[GitHub repository](master/example_scenes). You can also visit the [official gallery](https://docs.manim.community/en/stable/examples.html) for more advanced examples.
+[GitHub repository](https://github.com/ManimCommunity/manim/tree/master/example_scenes). You can also visit the [official gallery](https://docs.manim.community/en/stable/examples.html) for more advanced examples.
```
Comment (WampyCakes) [inline README.md:80]: I think that making it link to either `/example_scenes` or `example_scenes` (not sure which) would actually work. So essentially just removing `master`. You'll have to try it tho

Comment (nidhaloff) [reply README.md]: @WampyCakes thanks for the hint. You are right, changing it to just **example_scenes** works fine

**Review by jsonvillanueva — APPROVED** [review]
Thanks for the contribution!

**Review by behackl — APPROVED** [review]
LGTM, good job fixing the broken link! Thanks for contributing!

**Pull request merged**